src/Installers/Windows/AspNetCoreModule-Setup/ANCMIISExpressV1/AncmIISExpressV1.wixproj

```diff
@@ -56,17 +29,41 @@
   </ItemGroup>
 
   <ItemGroup>
+    <Compile Include="$(IIS-Setup)iisca\wix3\iisca.wxs">
+      <Link>iisca.wxs</Link>
+    </Compile>
+    <Compile Include="ancm_iis_express.wxs" />
     <EmbeddedResource Include="..\setupstrings.wxl">
       <Link>setupstrings.wxl</Link>
     </EmbeddedResource>
-  </ItemGroup>
-  <ItemGroup>
     <Content Include="$(IIS-Setup)include.wxi">
       <Link>include.wxi</Link>
     </Content>
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\CustomAction\aspnetcoreCA.vcxproj">
+      <Name>aspnetcoreCA</Name>
+      <Project>{7c27e72f-54d0-4820-8cfa-5e4be640974b}</Project>
```

Comment:
Is this still required?

Reply:
Not required for command line builds, only for opening this project in Visual Studio.